docs/griptape-framework/structures/prompt-drivers.md

```diff
@@ -53,6 +53,9 @@ print(result.value)
 
 Griptape offers the following Prompt Drivers for interacting with LLMs.
 
+!!! warning
+    When overriding a default Prompt Driver, take care to ensure the Structure's configured Embedding Driver is compatible with the Prompt Driver you've selected. If Task Memory isn't needed, you can avoid compatability issues by setting `task_memory=None` to disable Task Memory in your Structure.
```

Comment:
@andrewfrench "compatible" might not be the right word here. You _can_ use an OpenAi Embedding Driver with an Bedrock Prompt Driver, it's just that the user probably didn't mean to do so.

Reply:
Updated to use less urgent wording.